spikeextractors/extractors/jrcsortingextractor/jrcsortingextractor.py

```diff
@@ -73,14 +73,16 @@ def __init__(self, file_path: PathType, positive_only: bool = False):
             mask = (spike_clusters == uid)
             self._spike_trains[uid] = spike_times[mask]
 
-            self.set_unit_spike_features(uid, "spike_amplitudes", spike_amplitudes[mask])
-            self.set_unit_spike_features(uid, "spike_sites", spike_sites[mask])
-            self.set_unit_spike_features(uid, "spike_positions", spike_positions[mask, :])
+            self.set_unit_spike_features(uid, "amplitudes", spike_amplitudes[mask])
+            self.set_unit_spike_features(uid, "max_channel", spike_sites[mask])

```

Comment:
Sorry to bother again, maybe this can be max_channels for even more consistency? :)